lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python
+#-------------------------------------------------------------------------------
+# Name:        passwordstore.py
```

Comment:
The licensing should match the rest of Ansible, see [this example](https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/lookup/dig.py#L1-L16)